go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/ipfs/go-log v1.0.0 // indirect
 	github.com/libp2p/go-libp2p-core v0.3.0
 	github.com/minio/blake2b-simd v0.0.0-20160723061019-3f5f724cb5b1
+	github.com/multiformats/go-multiaddr v0.2.0
```

Comment:
See [this discussion](https://github.com/filecoin-project/specs-actors/issues/425#issuecomment-638521129) about why I really want to avoid this.